### PR TITLE
Adds the Announcement role to new users

### PR DESCRIPTION
**Pull request by Ramsfield**
To help mitigate a lot of the issues of global pings in discord. This will treat a role as special, giving it to everyone as they join, and the user can opt out of it if they so please.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -516,10 +518,9 @@ async def manage_reactions(payload, added: bool):
...
     botChannel = discord.utils.get(member.guild.channels, name='bot-stuff')
     rulesChannel = discord.utils.get(member.guild.channels, name='rules-and-info')
-    await botChannel.send((
-        f'Welcome to the server {member.mention}!\nPlease check out {rulesChannel.mention}!\nIn order to view channels you need to add the relevant roles.\n\
-        Type !help for help, !serverroles for the roles you can add yourself to, !add "role1" "role2" to put yourself in that course.'
-    ))
+    role = discord.utils.get(member.guild.roles, name=announcementChanName)
+    await member.add_roles(role)
+    await botChannel.send(f"""Welcome to the server {member.mention}!\nPlease check out {rulesChannel.mention}!\nIn order to view channels you need to add the relevant roles.\nType `!help` for help, `!serverroles` for the roles you can add yourself to, `!add role1 role2` to put yourself in that course.\nYou have already been added to the {announcementChanName} role, so that you can keep update on any events that might be happening and things you might want to be aware of. Feel free to remove yourself from this role by saying `!sub {announcementChanName}` in {botChannel.mention}""")
```
Comment (CodeMastr3) [inline bot.py:526]: I would prefer it if you could split this back up into the multiline send for easier readability

**Comment by CodeMastr3** [comment]
But other than my 1 comment looking very interesting and a nice idea

**Comment by Ramsfield** [comment]
@CodeMastr3 I'm more than happy to fix the multiline thing if you want. I just haven't had time yet

**Comment by CodeMastr3** [comment]
@Ramsfield You might be able to just add another commit onto this but I also just figured I could do it but the functionality was needed and just the style didn't matter too much

**Pull request merged**